internal/app/machined/pkg/system/services/kubelet.go

```diff
@@ -225,47 +232,99 @@ func (k *Kubelet) HealthSettings(runtime.Configurator) *health.Settings {
 	return &settings
 }
 
+func newKubeletConfiguration(clusterDNS []string) *kubeletconfig.KubeletConfiguration {
+	f := false
+	t := true
+
+	return &kubeletconfig.KubeletConfiguration{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "kubelet.config.k8s.io/v1beta1",
+			Kind:       "KubeletConfiguration",
+		},
+		StaticPodPath:      "/etc/kubernetes/manifests",
+		Address:            "0.0.0.0",
+		Port:               10250,
+		ReadOnlyPort:       10255, // TODO(andrewrynhard): Disable this.
```

Comment:
should this be disabled now or is that for future andrew?

Reply:
Future Andrew :D ... this is something we can do to improve security, but nothing we did prior to this PR.

Reply:
I should probably make a ticket.